Content.Server/GameObjects/Components/Observer/GhostRoleComponent.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Content.Server.GameObjects.EntitySystems;
+using Robust.Server.Interfaces.Player;
+using Robust.Shared.GameObjects;
+using Robust.Shared.GameObjects.Systems;
+using Robust.Shared.Serialization;
+
+namespace Content.Server.GameObjects.Components.Observer
+{
+    public abstract class GhostRoleComponent : Component
+    {
+        public string RoleName { get; set; }
+        public string RoleDescription { get; set; }
+        public bool Taken { get; set; }
```

Comment:
Does it make sense for these to be settable?

Reply:
Not really, I could make the setters private without issue